test/helper.js

```diff
@@ -24,8 +24,8 @@ const resetDatabase = async () => {
     DELETE FROM "Users";
     DELETE FROM "Programs";
     DELETE FROM "Locations";
+    DELETE FROM "ProgramLocations"
```

Comment:
The error you are getting is caused by you missing a semicolon at the end of this line.